.devcontainer/setup.py

```diff
@@ -105,7 +99,8 @@ def main():
     make_init()
 
     # do basic setup
-    setup()
+    # TODO: fix the setup and revert
+    # setup()
```

Comment:
This will be fixed in another PR?

Reply:
Ah, read PR description

Reply:
Yes, preferably, to let the devcontainer get built.